products/pubsub/terraform.yaml

```diff
@@ -124,6 +124,14 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         default_from_api: true
       expirationPolicy.ttl: !ruby/object:Overrides::Terraform::PropertyOverride
         diff_suppress_func: 'comparePubsubSubscriptionExpirationPolicy'
+      retryPolicy: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
```

Comment:
Just to confirm- if the user does not send a retry policy, one will still be returned by the API?